scripts/autorelease.sh

```diff
@@ -37,9 +37,11 @@ git push --force --tags
 # Combine all assets
 assets=()
 for file in $assets_dir/*; do
-    echo $file
-    assets+=("-a" $file)
+    assets+=($file)
 done
```

Comment:
Looks like you can simply pass `$assets_dir/*` to `gh release upload` call below

Reply:
:smart: